File: File_Line_Counter_with_Word_Stats.py
```python
import os

def count_lines(file_path):
    with open(file_path, 'r') as file:
        lines = file.readlines()
        return len(lines)

def count_words(file_path):
    with open(file_path, 'r') as file:
        words = file.read().split()
        return len(words)

def main():
    file_path = input("Enter file path: ")
    if not os.path.isfile(file_path):
        print("Invalid file path")
        return
    line_count = count_lines(file_path)
    word_count = count_words(file_path)
    print(f"Line count: {line_count}")
    print(f"Word count: {word_count}")

if __name__ == "__main__":
    main()
```

File: File_Line_Counter_with_Word_Stats.cs
```CSharp
using System;
using System.IO;
using System.Linq;

class Program
{
    static void Main(string[] args)
    {
        if (args.Length == 0)
        {
            Console.WriteLine("Please provide a file path.");
            return;
        }

        string filePath = args[0];

        if (!File.Exists(filePath))
        {
            Console.WriteLine("File does not exist.");
            return;
        }

        string[] lines = File.ReadAllLines(filePath);
        int lineCount = lines.Length;

        int wordCount = 0;
        foreach (string line in lines)
        {
            string[] words = line.Split(new char[] { ' ', '\t' }, StringSplitOptions.RemoveEmptyEntries);
            wordCount += words.Length;
        }

        Console.WriteLine("Line Count: " + lineCount);
        Console.WriteLine("Word Count: " + wordCount);
    }
}
```